common/scala/src/main/scala/whisk/core/entity/ActivationResult.scala

```diff
@@ -52,10 +52,10 @@ protected[core] object ActivationResponse extends DefaultJsonProtocol {
     /* The field name that is universally recognized as the marker of an error, from the application or otherwise. */
     val ERROR_FIELD: String = "error"
 
-    private val Success          = 0
-    private val ApplicationError = 1
-    private val ContainerError   = 2
-    private val WhiskError       = 3
+    val Success          = 0
+    val ApplicationError = 1
+    val ContainerError   = 2
+    val WhiskError       = 3
```

Comment:
We used these as constants to return (see https://github.com/openwhisk/openwhisk/pull/642/files#diff-2850946ee2d35de3b057e40f10f1887fR131 for example), so we wouldn't need magic numbers.